python/ray/tests/test_runtime_env.py

```diff
@@ -52,7 +52,7 @@ def test_get_wheel_filename():
 
 
 def test_get_master_wheel_url():
-    ray_version = "2.0.0rc0"
+    ray_version = "3.0.0.dev0"
     test_commit = "c3ac6fcf3fcc8cfe6930c9a820add0e187bff579"
```

Comment:
For this test to pass, the test_commit needs to be changed to a commit from 3.0.0.dev0

Here's an example which you can use, I checked that it has all the wheels built:
`d79431e32cffbf3f86da5f7417697dc8edd1da3f`
https://s3.console.aws.amazon.com/s3/buckets/ray-wheels?region=us-west-2&prefix=master/d79431e32cffbf3f86da5f7417697dc8edd1da3f/&showversions=false

Reply:
But better to run the test locally to be sure

Reply:
thanks!